src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -0,0 +1,59 @@
+package by.onliner.page;
+
+import by.onliner.util.MoveTo;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ShoppingCartPage extends AbstractPage {
+
+    @FindBy(xpath = "//div[contains(@class, 'cart-form__title')]")
+    WebElement openedShoppingCartMessage;
+
+    @FindBy(className = "cart-form__control")
+    WebElement removeFromCartButton;
+
+    @FindBy(xpath = "//div[contains(@class, 'cart-form__offers-unit_primary')]")
+    WebElement Item;
```

Comment:
please use proper naming for this element. Variable's name should start with a small letter and be more informative

Reply:
Renamed to cartUnit